lib/phlex/csv.rb

```diff
@@ -82,9 +82,13 @@ def render(renderable)
 	end
 
 	def escape(value)
-		value = value.to_s
+		value = value.to_s.dup
+		value.strip!
```

Comment:
This should be optional. Perhaps you should be able to define `def strip_values? = false` or something like that.